lxl-web/src/routes/(app)/[[lang=lang]]/(find)/[fnurgel=fnurgel]/InstancesList.svelte

```diff
@@ -110,18 +112,18 @@
 				{@const id = relativizeUrl(getResourceId(item))}
 				<li {id} class="border-t border-t-primary/16">
 					<details
-						{...id && {
-							open:
-								$page.state.expandedInstances?.includes(id) ||
-								$page.url.searchParams.getAll('expanded').includes(id) ||
-								data.length === 1
-						}}
+						open={$page.state.expandedInstances?.includes(id) ||
+							$page.url.searchParams.getAll('expanded').includes(id) ||
+							data.length === 1}
 						on:toggle={() => handleToggleDetails($page.state)}
 					>
 						<summary
-							class="flex min-h-11 gap-2 pl-2 align-middle hover:bg-pill/16"
+							class="flex min-h-11 gap-2 pl-1 align-middle hover:bg-pill/16"
 							on:keydown={handleSummaryKeydown}
 						>
+							<span class="arrow flex items-center pr-2">
```

Comment:
I suggest removing the `pr-2` on the arrow (and adding space elsewhere), it becomes a padding-bottom when it rotates and results in a skewed rotation (not around centre)

Reply:
I would also add `w-fit` and `h-fit` so the span size will be equal to the size of the SVG. See https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content

`items-center` can then be moved to the parent `summary` element.